UnityProject/Assets/Scripts/Systems/Construction/MachinePartsItemTraits.cs

```diff
@@ -50,5 +50,35 @@ public class MachinePartsItemTraits : SingletonScriptableObject<MachinePartsItem
 		public ItemTrait SuperCapacityPowerCell;
 		public ItemTrait HyperCapacityPowerCell;
 		public ItemTrait BluespacePowerCell;
+
+		public bool IsComponent(ItemTrait ItemTrait)
```

Comment:
similar to how ``Tier`` is part of the trait, maybe this bool should also be part of it.

Reply:
this is only needed for legacy support